metapool/tests/test_amplipool.py

```diff
@@ -0,0 +1,273 @@
+import pandas as pd
+
+from unittest import TestCase, main
+from metapool.amplipool import assign_emp_index, _load_emp_indices
+
+
+class AmplipoolTests(TestCase):
+    def setUp(self):
+        self.plate_metadata = pd.DataFrame([
+            {
+                'Plate Position': '1',
+                'Primer Plate #': '1',
+                'Plating': 'SF',
+                'Extraction Kit Lot': '166032128',
+                'Extraction Robot': 'Carmen_HOWE_KF3',
+                'TM1000 8 Tool': '109379Z',
+                'Primer Date': '2021-08-17',
+                'MasterMix Lot': '978215',
+                'Water Lot': 'RNBJ0628',
+                'Processing Robot': 'Echo550',
+                'Sample Plate': 'THDMI_UK_Plate_2',
+                'Project_Name': 'THDMI UK',
+                'Original Name': ''
```

Comment:
Any reason why in the tests 'Original Name' is always ''?

Reply:
Keeping empty for now, but we want to check this with Gail so we know if we should remove it or what value should be added instead.